### DISCUSSION TITLE
Emulating Bold Font

### DISCUSSION THREAD

**Discussion by tbmuc** [ImageMagick/ImageMagick#2338] [Help] [answered]
Hello,

a specific custom font is only available as regular version. In Photoshop it is possible to emulate bold appearance by "Faux Bold". Is there a way to do the same in IM 7 (Windows 10)?

**Comment by fmw42** [comment]
Perhaps use font-weight bold. See https://imagemagick.org/script/magick-vector-graphics.php in -draw

**Comment by snibgo** [comment, marked as answer]
You could write your string multiple times with offsets, for example:
```
magick -size 600x200 xc: -pointsize 75 -gravity NorthWest -annotate +0+0 "Hello world" -annotate +5+0 "Hello world" -trim out.png
```
![x](https://user-images.githubusercontent.com/33812368/88836328-6bc8a080-d1ce-11ea-9b51-d50aa3baf0ea.png)

**Reply by tbmuc** [reply]
Great, that works.

**Comment by GeeMack** [comment]
Depending on the pointsize and finished image size, etc. you may be able to use "-morphology" to expand the lettering a bit. You probably need to use a transparent background. Here are example commands...

    magick -pointsize 72 -background none label:"Testing 1 2 3" result1.png

    magick -pointsize 72 -background none label:"Testing 1 2 3" -morphology dilate square result2.png

These are the output images...

![result1](https://user-images.githubusercontent.com/65475279/88838778-a7984180-d19f-11ea-84b6-60b499c834e5.png)

![result2](https://user-images.githubusercontent.com/65475279/88838793-acf58c00-d19f-11ea-945d-a9ebedbb8550.png)

Try various morphology kernels like "square" or "octagon" to create more or less dilation.
